release_templates/capi/capo/>=v0.6.0/cluster_config.yaml

```diff
@@ -0,0 +1,41 @@
+bastion:
+  bootFromVolume: true
+  diskSize: 10
+  flavor: n1.tiny
+  image: aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
```

Comment:
I don't quite get this one - what is the reason for a base where you need to then overlay 99% of the variables? From an end user perspective that seems not very helpful to me.

Reply:
Like that is doesn't have much sense, I agree. As part of the `gitops-template` it makes sense, because image uuid there serves only a demo purpose there, putting it here makes an impression it's something user can use "off the shelf"

Reply:
that's a very good point @MarcelMue, for CAPO at least since the UUIDs, AZs, etc., depends on the project it may not make sense to put bases in here at all